core/src/main/java/com/dooapp/fxform/validation/PropertyElementValidator.java

```diff
@@ -50,7 +48,21 @@ public class PropertyElementValidator {
     public PropertyElementValidator(final PropertyElement element) {
         this.element = element;
         validator.addListener((observableValue, validator, validator2) -> validate(element.getValue()));
-        invalid.bind(Bindings.isNotEmpty(constraintViolations));
+        invalid.bind(new BooleanBinding() {
+            {
+                super.bind(constraintViolations);
+                super.bind(constraintViolations.get());
```

Comment:
Just a warning here.

Standard bindings does not work like the projet FXBindings which reconfigure dependencies when one of them become invalid (and allow to easily change the ObservableList inside a ListProperty).
So this second binding is not necessary here, it will just trigger more time the method computeValue() when the content list will change, because both ListProperty et the included ObservableList will trigger computeValue().

So this second super.bind is unnecessary here, and constraintViolations.get() in computeValue() can just be constraintViolations.